src/main/kotlin/lotto/LottoMachine.kt

```diff
@@ -1,20 +1,19 @@
 package lotto
 
 class LottoMachine() {
-    fun createTickets(userAmount: Int): List<Lotto> {
+    fun createGeneratedTickets(
+        totalPurchasedTickets: Int,
+        manualTicketsQuantity: Int,
+    ): List<Lotto> {
         val lottos = mutableListOf<Lotto>()
-        val amountOfTickets = calculateTickets(userAmount)
-        var count = 0
-        while (count < amountOfTickets) {
+        var ticketsToBeGenerated = totalPurchasedTickets - manualTicketsQuantity
+        var countGeneratedTickets = 0
+        while (countGeneratedTickets < ticketsToBeGenerated) {
             lottos.add(Lotto.create())
-            count++
+            countGeneratedTickets++
```

Comment:
```Kotlin
repeat(ticketsToBeGenerated) {
    lottos.add(Lotto.create())
}
```

Same applies to for, loop, or repeat operations as well.
We don't have to count unless there's a chance for we to modify it or get a different outcome.